### PR TITLE
improved the passphrase form behaviour

### PR DESCRIPTION
**Pull request by maarteNNNN**

### CODE REVIEW

File: src/views/Login.vue [hunk 1]
```diff
@@ -166,17 +168,22 @@ export default {
...
             } catch (e) {}
 
             lastInput.focus();
+
+            pasting.value = false;
           } else if (element && element.includes(' ')) {
             const nextInput = document.getElementById(`passphrase-${i + 1}`);
             if (nextInput) {
               nextInput.focus();
+              setTimeout(() => (nextInput.value = ''), 50);
```
Comment (jondubois) [inline src/views/Login.vue:186]: Why does it require a 50 millisecond timeout?

Comment (jondubois) [reply src/views/Login.vue:186]: Can this be `0`? Instead of `50`. Usually to get something to run on the 'next tick', setTimeout 0 is used.

Comment (maarteNNNN) [reply src/views/Login.vue]: See commit 795f2a60ae9240f2b9e4d8a770f3d3785717d5d7 where I used Vue's `nextTick` solution

**Comment by maarteNNNN** [comment]
@jondubois This is ready to go

**Review by jondubois — APPROVED** [review]

**Pull request merged**